app/main/index.ts

```diff
@@ -261,6 +264,10 @@ app.on('ready', () => {
 		page.send('toggle-autohide-menubar', showMenubar, true);
 	});
 
+	ipcMain.on('fix-bounds', () => {
```

Comment:
Needs to document this.

Reply:
Will do after the testing and bug fixing is done.

Reply:
DOne.